core/main_test.ts

```diff
@@ -617,7 +617,65 @@ actions:
             }),
             target: {
               database: "dataform",
-              name: "note"
+              name: "notebook"
+            }
+          }
+        ])
+      );
+    });
+  });
+
+  suite("SQL actions", () => {
```

Comment:
another test for a non-operation failing

Reply:
My next PRs will add support for the other action types so I'm not bothering